src/element-style-observer.js

```diff
@@ -5,13 +5,14 @@ import { toArray, wait, getTimesFor } from "./util.js";
 import RenderedObserver from "./rendered-observer.js";
 
 // Start bug detection early if we have a DOM
+let allowDiscrete = "";
 if (globalThis.document) {
-	Bugs.detect();
-}
+	Bugs.detectAll();
 
-const allowDiscrete = globalThis.CSS?.supports?.("transition-behavior", "allow-discrete")
-	? " allow-discrete"
-	: "";
```

Comment:
@LeaVerou – I don't understand why you moved this inside `if (globalThis.document)`; is there a case where we would have `globalThis.CSS.supports` but no `globalThis.document`?

Reply:
Just because it seems it's not relevant if we have no document.